homeassistant/components/ezviz/sensor.py

```diff
@@ -36,6 +36,8 @@
         key="PIR_Status",
         device_class=DEVICE_CLASS_MOTION,
     ),
+    "last_alarm_type_code": SensorEntityDescription(key="last_alarm_type_code"),
+    "last_alarm_type_name": SensorEntityDescription(key="last_alarm_type_name"),
```

Comment:
These sensors should probably be categorized as diagnostic. Can be done in a follow up.